lib/df-detection.c

```diff
@@ -67,7 +67,8 @@ void report(void)
 	pr_err("BUG: Intersection Detected \n ");
```

Comment:
Note that syzkaller treats whatever you write after "BUG:" as a report header, and it clusters together reports with similar headers.
So we probably want to at least print the syscall function name on the same line - this way we'll get different different syscalls bucketed separately.
In the future we'll probably want to go even further and add the call sites to the header to separate bugs from each other.